### DISCUSSION TITLE
Microsoft.Azure.Functions.Worker.Sdk 1.17.3 makes Function completely unresponsive in Azure.

### DISCUSSION THREAD

**Discussion by ticedev-1972** [Azure/azure-functions-dotnet-worker#2607] [Feedback]
Microsoft.Azure.Functions.Worker.Sdk 1.17.3 (and 1.17.4) makes Function completely unresponsive in Azure. Nothing is logged in application insights. Everything just works locally. When I downgrade to 1.17.2 it works on Azure again.
I won't be able to share any code or logs here for security reasons. Just curious if anyone has a similar experience ?

**Comment by EdwinOtten** [comment]
I created an issue for the same problem: #2609.
OP is a colleage of mine on the same team, so until now we are the only ones report this issue.
